src/promptflow-devkit/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -258,6 +260,11 @@ def get_pfs_host():
     return pfs_host
 
 
+def get_pfs_host_after_check_wildcard(service_host):
+    """When the address is 0.0.0.0, return the default host."""
```

Comment:
may add reason why we need to apply such a mapping - i understand it's because we need to ping 127.0.0.1:port for heartbeat, instead of 0.0.0.0:port?

Reply:
or just update this in the PR description, so that we can find the reason somewhere

Reply:
yeah, pfs can hold on 0.0.0.0. But need ping using valid ipv4 ip address